### PR TITLE
Minor fix in dev packages, do not put core-dev in the official opam index

### PR DESCRIPTION
**Pull request by mattam82**
Solves issues #99 and #182

### CODE REVIEW

**Comment by mattam82** [comment]
@gares Are you ok with both fixes?

File: core-dev/packages/coq.8.7.dev/opam [hunk 1]
```diff
@@ -12,8 +12,7 @@ build: [
     "-prefix" prefix
     "-usecamlp5"
     "-camlp5dir" "%{lib}%/camlp5"
-    "-coqide" "no"
-    "-debug"]
+    "-coqide" "no"]
```
Comment (silene) [inline core-dev/packages/coq.8.7.dev/opam]: The official Coq package is compiled with `-debug`. Assuming that is correct, there is no reason for the unofficial ones to be compiled without it.

Comment (mattam82) [reply core-dev/packages/coq.8.7.dev/opam]: It's now the default

**Comment by gares** [comment]
yes

**Pull request merged**